pkg/agent/agent.go

```diff
@@ -116,8 +118,8 @@ type Flows struct {
 	rbTracer  *flow.RingBufTracer
 	accounter *flow.Accounter
 	limiter   *flow.CapacityLimiter
-	deduper   node.MiddleFunc[[]*flow.Record, []*flow.Record]
-	exporter  node.TerminalFunc[[]*flow.Record]
+	deduper   node.MiddleFunc[[]*model.Record, []*model.Record]
+	exporter  node.TerminalFunc[[]*model.Record]
 
 	// elements used to decorate flows with extra information
 	interfaceNamer flow.InterfaceNamer
```

Comment:
does this need to be model.InferfaceNamer

Reply:
I guess we could move that to model, although it won't make a big difference in the grand scheme of dependencies :-)